pkgs/top-level/python-aliases.nix

```
@@ -491,6 +491,7 @@ mapAliases ({
   ntlm-auth = throw "ntlm-auth has been removed, because it relies on the md4 implementation provided by openssl. Use pyspnego instead.";
   oauth = throw "oauth has been removed as it is unmaintained"; # added 2025-05-16
   oauth2 = throw "oauth2 has been removed as it is unmaintained"; # added 2025-05-16
+  objax = throw "objax has been removed because it has been marked as broken since at least November 2024."; # Added 2025-10-04
```

```suggestion
  objax = throw "objax has been removed because the upstream project was archived."; # Added 2025-10-04
```
Another reason for removing it is that it is actually archived.